File: executable/julia/privacy-true.jl
using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

data = [
  [
    "x" => [0, 2, 4], 
    "y" => [0, 4, 2], 
    "type" => "scatter"
  ]
]

response = Plotly.plot([data], ["filename" => "privacy-true", "fileopt" => "overwrite"])
plot_url = response["url"]

File: executable/r/privacy-true.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

data <- list(
  list(
    x = c(0, 2, 4), 
    y = c(0, 4, 2), 
    type = "scatter"
  )
)

response <- p$plotly(data, kwargs=list(filename="privacy-true", fileopt="overwrite", world_readable="TRUE"))
url <- response$url
filename <- response$filename